docs/reference/how-to/search-speed.asciidoc

```diff
@@ -9,6 +9,27 @@ fast. In general, you should make sure that at least half the available memory
 goes to the filesystem cache so that Elasticsearch can keep hot regions of the
 index in physical memory.
 
+[discrete]
+=== Avoid page cache thrashing by using modest read ahead values on Linux
+
+Search can cause a lot of randomized read IO. When using the default
+Elasticsearch filestore (fs/<<hybridfs,hybridfs>>), Lucene files using memory
+mapping may contribute to unnecessary read I/O when the underlying block device
+has a high readahead value.
+
+Most Linux distributions use a sensible readahead value of `128KiB` for a
+single plain device, however, when using software raid, LVM or dm-crypt the
+resulting block device (backing Elasticsearch <<path-settings,path.data>>)
+may end up having a very large readahead value (in the range of several MiB).
+This usually results in severe page (filesystem) cache thrashing adversely
```

Comment:
I think the term "page cache" is established enough by now so we can remove `(filesystem)`?

Reply:
Good point; unfortunately the term `filesystem cache` is used rather consistently in this doc instead of page cache (e.g. [here](https://github.com/elastic/elasticsearch/blame/227018225f2f70c148e4394c8baddd4176585356/docs/reference/how-to/search-speed.asciidoc#L380)). Therefore I wanted to use the more familiar to systems folks "page cache" term, but still have some connection with the rest of the doc.

Reply:
I see, that works for me then but I think we should get rid of the old name eventually in our docs.